target/linux/bcm63xx/base-files/etc/uci-defaults/09_fix_crc

```diff
@@ -33,6 +33,7 @@ case "$(board_name)" in
 	nucom,r5010un-v2|\
 	observa,vh4032n|\
 	t-com,speedport-w-303v|\
+	technicolor,tg582n-telecom-italia|\
```

Comment:
Please, add a proper switch configuration (with VLANs).
If the none of the 4 ethernet ports is labelled as WAN, then the 4 ethernet ports should be added to eth0.1.

Reply:
done